app/frontend/wallet/cardano-trezor-crypto-provider.js

```diff
@@ -14,6 +12,30 @@ const CardanoTrezorCryptoProvider = (ADALITE_CONFIG, walletState) => {
     throw new Error(`Unsupported derivation scheme: ${state.derivationScheme.type}`)
   }
 
+  const deriveXpub = generateDeriveXpubFn(async (absDerivationPath) => {
+    const response = await TrezorConnect.cardanoGetPublicKey({
+      path: absDerivationPath,
+      showOnTrezor: false,
+    })
+    if (response.error || !response.success) {
+      return Promise.reject(response || 'operation failed')
```

Comment:
Why not throw instead?

Reply:
if you just do `throw new Error(response || 'operation failed'), the "response" object gets transformed to string "[object Object]" and when it's printed to the console, the information about the exact response is lost. But I deem this redundant and the failure should be debuggable even without explicitly returning the response, so I replaced it with a throw